data/bbb-h264-all-i-32/frames.h

```diff
@@ -27,14 +27,16 @@
 				.flags = 0 | 0 | V4L2_H264_SPS_FLAG_FRAME_MBS_ONLY | 0 | V4L2_H264_SPS_FLAG_DIRECT_8X8_INFERENCE,
 			},
 			.scaling_matrix = {
-				.scaling_list_4x4 = {
+				/* 4x4[6][16] */
```

Comment:
Since these headers are generated with libva-dump, it is not really relevant to add documentation there: we'd like to keep them as they are produced by the tool.

Reply:
I didn't know that. Of corse it makes sense to use defaults from libva-dump.